.github/CONTRIBUTOR_GUIDE/ct/AppName.sh

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
```

Comment:
This is nearly what I suggested with "the documented version" (see above).

I think this file should not be here as a single file, but included in the markdown file.

Reply:
In my opininon this is easier thio grasp for a new contributor, see my comment above.